### DISCUSSION TITLE
Illustration resizing

### DISCUSSION THREAD

**Discussion by corinne-nagel** [twilio-labs/paste#1644] [Request: Illustration] [answered]
Hi team, Riley suggested there's a new illustration process and asked me to reach out about resizing. Please let me know if there's a different process for this. I'm unable to attach the files here even after zipping them so lmk who will take this and I can send via Slack.

Thanks!

**Comment by kate-porter** [comment]
Hey @corinne-nagel! Are the illustrations in Bynder? If so, you can just post the Bynder links here and we can download the files. If they aren't in Bynder, can you ask Riley to add them? Theoretically, all illustrations should be there.

**Reply by corinne-nagel** [reply]
Here ya go!
https://library.twilio.com/share/E5F439F6-5933-4014-989E3F7897AA3CF9/
https://library.twilio.com/share/D83FF10C-F452-4E1D-95574B633209BDDF/
https://library.twilio.com/share/B967C48C-2BFA-46F2-95DE631EA180203D/
https://library.twilio.com/share/14F633F2-AB77-4301-A0F9F13E66FEF2E9/

I think you're already working on this, but it seems like we're all re-using the same ones over and over and it's making for a bit of a stale experience. Would love to figure out how we can make it easier to get them requested/added as I went through the process recently and it was 🙅‍♀️  Let me know if I can help.

Thanks!

**Reply by kate-porter** [reply]
Thank you! I just created a [ticket](https://issues.corp.twilio.com/browse/DSYS-2866) to get these optimized for product and added to Bynder. When do you need the illos?

And we are not working on the repetitive illustration situation, actually. Brand really owns the creation of the illustrations, and we just handle optimizing them to be used as SVGs in product. I'd recommend chatting with them about getting more illustrations added, if you haven't already!

**Reply by corinne-nagel** [reply]
Probably by the end of this sprint if possible. And my bad, when she said you were taking over the process I got too excited. I guess she just meant the optimizing process lol 😢

**Reply by kate-porter** [reply]
Haha, yeah, sorry to disappoint! Maybe we'll have a dedicated illustrator for product someday, but we're not quite there yet. 😂

And cool, we can aim to pull that ticket into our next sprint (starting next Wednesday) to get them to you by EOM.

**Comment by kelseycavitt** [comment, marked as answer]
@corinne-nagel
Hey Corrine, I just joined the DSys team and was able to work on the illustrations you need. The SVGs have been added to a Bynder collection called [Product Optimized Illustrations.](https://library.twilio.com/collections/view/5E25D9F0-00FF-443B-9F8F395FDE06AEBC/) They have also been updated in the [Brand Illustrations Figma file](https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations?node-id=1%3A48). Let us know if you have any further questions! Thanks!

**Reply by corinne-nagel** [reply]
Cool cool, I'm not seeing some of the ones I needed in the Figma but I'll just pull them from Bynder. Thanks!

**Reply by kelseycavitt** [reply]
Oh, which ones are you not seeing in Figma? There are a few pages in the Figma file too, so you may have to pop around. Let me know!
